API/Controllers/ReviewsController.cs

```diff
@@ -23,16 +23,130 @@ public ReviewsController(IElasticClient elasticClient)
 
         }
 
-        [HttpGet("get10")] //api/movies
-        public async Task<ActionResult<List<Reviews>>> GetReviews()
+        [HttpGet("get10")] //api/reviews
+        public async Task<ActionResult<List<Review>>> GetReviews()
         {
-            var response = await _elasticClient.SearchAsync<Reviews>(s => s
+            var response = await _elasticClient.SearchAsync<Review>(s => s
                 .Index(reviewIndex)
                 .Query(q => q.MatchAll()));
-            // returns all movies (actually defaults to first 10)
+            // returns all reviews (actually defaults to first 10)
 
             return response.Documents.ToList();
         }
+
+        [HttpGet("getall (DO NOT CALL)")] //api/reviews   Should not call. Included just in case but dude. theres ~50k reviews
```

Comment:
I left it there for now just in case it's needed for something on the frontend, but I intend to implement some kind of pagination/chunking of the results later so that the function is more realistic to use.